src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -469,4 +469,7 @@
   <data name="MinimizeToTrayCommandKey" xml:space="preserve">
     <value>Minimize current window to tray</value>
   </data>
+	<data name="QuitCommandKey" xml:space="preserve">

```

Comment:
these are tabs, but I'm gonna be real, I'm the one who usually ends up reformatting this file in much worse ways